features/step_definitions/settings_steps.rb

```diff
@@ -50,3 +50,13 @@
     assert input.readonly?
   end
 end
+
+Then "I should see deployment option {string} checked" do |option|
+  option_checked = find("[name='service[deployment_option]'][checked]").value
+
+  if option == 'APIcast'
+    assert option_checked == 'hosted'
+  else
+    assert option_checked == 'self_managed'
```

Comment:
Doesn't it make more sense to be `assert option_checked == option`? You can make more clear about available options by having step `Then "I should see deployment option hosted/self_managed checked"`.

Reply:
But all other tests are using the convention 'APIcast' / ""APIcast self-managed", instead of "hosted" / "self_managed".
I'm trying to follow the same convention